api/v1alpha1/keyvault_types.go

```diff
@@ -36,6 +36,7 @@ type AccessPolicyEntry struct {
 	TenantID string `json:"tenantID,omitempty"`
 	// ClientID - The client ID of a user, service principal or security group in the Azure Active Directory tenant for the vault. The client ID must be unique for the list of access policies.
 	ClientID string `json:"clientID,omitempty"`
+	ObjectID string `json:"objectID,omitempty"`
```

Comment:
minor: Missing documentation - I find the names of all of these AAD-esque fields quite obtuse so we should try to be clear about what's what (like you've done for the other fields here)

Reply:
Yep, will address this as well